project/nef.xcodeproj/xcshareddata/xcschemes/CarbonPage.xcscheme

```diff
@@ -52,7 +52,7 @@
       </BuildableProductRunnable>
       <CommandLineArguments>
          <CommandLineArgument
-            argument = "--page &quot;~/Desktop/BowPlayground.app/Contents/MacOS/BowPlayground.playground/Pages/adios.xcplaygroundpage&quot;"
+            argument = "--page &quot;/Users/miguelangel/Desktop/BowPlayground.app/Contents/MacOS/BowPlayground.playground/Pages/page.xcplaygroundpage&quot;"
             isEnabled = "YES">
```

Comment:
Personal information should not be committed to the repository. Even the original content should be removed, as it is dependent on your computer.

Reply:
Yep, I will care about it, I do not worry about it during the migration, but I think it the time to remove all of them, thanks ^^